src/Arcus.EventGrid.Tests.Unit/Publishing/EventGridPublisherBuilderTests.cs

```diff
@@ -25,161 +26,41 @@ public void ForTopic_NullOrEmptyEndpointTopic_ShouldFailWithArgumentException(st
         [InlineData(" ")]
         [InlineData("  ")]
         public void UsingAuthenticationKey_NullOrEmptyEndpointKey_ShouldFailWithArgumentException(string authenticationKey)
-        {
```

Comment:
Hmm, I think something was wrong merged here. All the tests are removed.

Reply:
Will revert